src/utils/url/getAbsoluteUrl.ts

```diff
@@ -2,8 +2,8 @@
  * Returns an absolute URL based on the given path.
  */
 export function getAbsoluteUrl(path: string, baseUrl?: string): string {
-  // Ignore absolute URLs.
-  if (!path.startsWith('/')) {
+  // Ignore Full URL
```

Comment:
There's no such thing as "full" URL 😉 There are relative and absolute URLs.

Reply:
sorry for my bad comment

Reply:
No worries!